token-lending/program/src/processor.rs

```diff
@@ -1026,13 +1047,7 @@ fn process_deposit_reserve_liquidity_and_obligation_collateral(
         clock,
         token_program_id,
     )?;
-    _refresh_reserve(
-        program_id,
-        reserve_info,
-        pyth_price_info,
-        switchboard_feed_info,
-        clock,
-    )?;
+    _refresh_reserve_lite(program_id, reserve_info, clock)?;
```

Comment:
One side effect I can think of is the `allowedBorrowValue` on the Obligation would be slightly off. Are we still reading this to display borrow power on the frontend? @0xodia

Reply:
yea, until the reserve is refreshed the legit way i believe bc the value of the obligation isn't updated with a new price

Reply:
allowed borrow value would still be correct when actually borrowing because that requires a full reserve refresh

Reply:
yeah this is still heavily used in FE as the borrow limit. The use is only visual though so not the worse thing if it's slightly off from a stale price (if i'm understanding this PR correctly)